quarkchain/cluster/tests/test_grpc_client.py

```diff
@@ -8,15 +8,15 @@
 
 
 class NormalServer(grpc_pb2_grpc.ClusterSlaveServicer):
-    def SetRootChainConfirmedBlock(self, request, context):
-        return grpc_pb2.SetRootChainConfirmedBlockResponse(
+    def AddRootBlock(self, request, context):
```

Comment:
same as before, prefer adding functions / tests instead of replacing ones (because in the future we may use `SetRootChainConfirmedBlock`

Reply:
Done.